### PR TITLE
Rails 6.0

### PR DESCRIPTION
**Pull request by NickLaMuro**
Links
-----

* Epic:  https://github.com/ManageIQ/manageiq/issues/19977
* Current Cross Repo: https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/179

### CODE REVIEW

File: manageiq-postgres_ha_admin.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "activesupport",     "~> 5.0"
```
Comment (bdunne) [inline manageiq-postgres_ha_admin.gemspec:23]: Can we apply https://github.com/ManageIQ/activerecord-id_regions/pull/18/files#r537735807 here as well?

Comment (NickLaMuro) [reply manageiq-postgres_ha_admin.gemspec:23]: Yup, agreed.  Let me get that done here as well.

**Comment by bdunne** [comment]
Released v3.1.2 with these changes

**Comment by NickLaMuro** [comment]
Kicked off a cross repo build with this change in place and the dependency on this PR removed:

- https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/179
- https://travis-ci.com/github/ManageIQ/manageiq-cross_repo-tests/builds/210133580

Hopefully it still passes 🤞

**Pull request merged**